pg_lakehouse/src/datafusion/context.rs

```diff
@@ -44,11 +42,11 @@ impl ContextProvider for QueryContext {
     }
 
     fn get_function_meta(&self, name: &str) -> Option<Arc<ScalarUDF>> {
-        Session::with_session_context(|context| {
-            let context_res = context.udf(name);
-            Box::pin(async move { Ok(context_res?) })
-        })
-        .ok()
+        let context = Session::session_context().unwrap_or_else(|err| {
+            panic!("{:?}", err);
+        });
```

Comment:
Why this instead of just `unwrap()`?

Reply:
Just so the error message is cleaner, doesn't have `unwrap` in it